docs/source/workflow_inputs.rst

```diff
@@ -858,6 +859,19 @@ If electric resistance heating is specified, additional information is entered i
   ``AnnualHeatingEfficiency[Units="Percent"]/Value``  double  frac   0 - 1        No        1.0      Efficiency
   ==================================================  ======  =====  ===========  ========  =======  ==========
 
+PTAC - Heating
+~~~~~~~~~~~~~~
+
+If electric resistance heating is specified, additional information is entered in ``HeatingSystem``.
+
+  ==================================================  ======  =========  ===========  ========  =======  ==================================
+  Element                                             Type    Units      Constraints  Required  Default  Notes
+  ==================================================  ======  =========  ===========  ========  =======  ==================================
+  ``AnnualHeatingEfficiency[Units="Percent"]/Value``  double  frac       0 - 1        No        1.0      Efficiency
+  ``AttachedToCoolingSystem``                         idref   See [#]_                No                 ID of attached ptac cooling system
+  ==================================================  ======  =========  ===========  ========  =======  ==================================
+
+  .. [#] AttachedToCoolingSystem is required for ``PackagedTerminalAirConditionerHeating``. The cooling system being attached must be of type "packaged terminal air conditioner".
```

Comment:
This is out of date because we're no longer using `AttachedToCoolingSystem`, right?